database/factories/TaskAttachmentFactory.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+/* @var $factory \Illuminate\Database\Eloquent\Factory */
+
+use App\Models\Task;
+use Faker\Generator as Faker;
+
+$factory->define(\App\Models\TaskAttachment::class, function (Faker $faker) {
+    $task = factory(Task::class)->create();
+
+    return [
+        'task_id' => $task->id,
```

Comment:
param `file` also important for task attachment, so remove this factory as of now or just add file here, and remove all tests of task attachment, we can cover it later